Header header
bool is_forward
bool is_cw
float32 linear_speed
float32 angular_speed